spec/interactors/omniauth/google/build_auth_client_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+describe Omniauth::Google::BuildAuthClient do
+  subject(:context) { described_class.call(auth_code: auth_code) }
+
+  let(:expected_attributes) do
+    {
+      access_token: nil,
+      access_type: :offline,
+      additional_parameters: {},
+      authorization_uri: instance_of(Addressable::URI),
+      client_id: "google_client_id",
+      client_secret: "google_client_secret",
+      code: "random_authorization_code",
+      expires_at: nil,
+      expiry: 60,
+      extension_parameters: {},
+      id_token: nil,
+      issued_at: nil,
+      issuer: nil,
+      password: nil,
+      principal: nil,
+      redirect_uri: instance_of(Addressable::URI),
+      refresh_token: nil,
+      scope: nil,
+      state: nil,
+      target_audience: nil,
+      token_credential_uri: instance_of(Addressable::URI),
+      username: nil
+    }
+  end
+  let(:expected_authorization_uri) do
+    "https://accounts.google.com/o/oauth2/auth?" \
+      "access_type=offline&" \
+      "client_id=google_client_id&" \
+      "redirect_uri=postmessage&" \
+      "response_type=code"
+  end
```

Comment:
Is it possible to define this variable as instance of `Addressable::URI`? For example:
```suggestion
  let(:expected_authorization_uri) do
    Addressable::URI.parse("https://accounts.google.com/o/oauth2/auth?" \
      "access_type=offline&" \
      "client_id=google_client_id&" \
      "redirect_uri=postmessage&" \
      "response_type=code")
  end
```
Then we can include it to `expected_attributes`:
```ruby
authorization_uri: expected_authorization_uri
```
and decrease amount of expectation below. What do you think about this?